### ISSUE TITLE
Did your readme miss the step create-nodepool.sh?

### ISSUE THREAD

**Issue by ShiqianTao** [nzimerman-varonis/aks-many-services-bug#1]
Did your readme miss the step `create-nodepool.sh`? (from Microsoft)

**Comment by ShiqianTao** [comment]
I may have found a key point: you have 3000+ services. It may be too heavy for HNS Policy.
I will double check the capacity of HNS Policy.

**Comment by trianglee** [comment]
@ShiqianTao The problem *is* reproduced without Overlay CNI as well.
Our setup calls for many more services (around 50,000). If indeed AKS on Windows cannot cope with such a setup, we definitely need to know.

**Comment by ShiqianTao** [comment]
I see, I reproduced the issue according to this repo. 
50000 services is a big number, I am checking with the Windows team and Upstream team.

**Comment by trianglee** [comment]
@ShiqianTao Yes, the plan was to use CNI Overlay. But we also tried without Overlay.
(you can reproduce without Overlay by changing "NETWORK_PLUGIN_MODE" in config.sh to be an empty string).
